hpi-swt2/lecture-portal
# Feature/#72 question upvote sorting

# Resolves #72 

This PR adds the feature of having questions sorted by either creation time or upvotes

Acceptance Criteria:
- [ ] Students and lecturers have access to this feature
- [ ] Questions are reordered in soft real-time
- [ ] I can select whether I want questions to be ordered by time or upvotes (see mockup)
- [ ] Default order for lecturer: Sort by votes
- [ ] Default order for student: Sort by time

Testing steps:
- as a student, go to "Questions" (/questions) and see questions sorted by creation time (and also create new questions to test)
- as a lecturer, go to "Questions" (/questions) and see question sorted by upvotes of students
- as either lecturer or student, try to change sorting by clicking on sort criteria
- rspec

Steps needed for migration:
- none

---

Definition Of Done:
